features/steps/table.py

```diff
@@ -351,6 +366,15 @@ def then_the_reported_column_width_is_width_emu(context, width_emu):
     )
 
 
+@then('the reported vertical alignment of the cell is {vAlign}')
+def then_the_reported_vertical_alignment_of_the_cell_is_vAlign(context, vAlign):
+    expected_vAlign = {'None': None, 'bottom': WD_VERTICAL_ALIGNMENT.BOTTOM}[vAlign]
+    actual_vAlign = context.cell.vAlign
+    assert actual_vAlign == expected_vAlign, (
+        'expected %s, got %s' % (expected_vAlign, actual_vAlign)
+    )
+
+
```

Comment:
Generally in these roles I just use the word "value" (rather than 'vAlign' in this case). There are a number of reasons, but one is that it can be consistent across different tests so makes it easier to read and make sense of quickly. Also, like in this case, if you have to change the name for some reason these don't have to change :)